src/detail-page/components/TabJobContent/FetchJobElement.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
```

Comment:
This import seems redundant (as far as I can see we are not using anything from the `react` library in this component).

Reply:
Yes, but I need this because i use jsx here ('React' must be in scope when using JSX - i get this message)

Reply:
Actually, we no longer need to have React in scope :) we have a new TypeScript version which enables us to use JSX without explicitly importing React.

You can read more about it here: https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html

In short, we have Typescript 4.1+ in the project, meaning we get new JSX transform by default. You can check it in tsconfig file - the "jsx" prop is now "jsx-react" which means we have the new JSX transform:
![image](https://user-images.githubusercontent.com/8350925/114260481-6d8d6980-99d5-11eb-8813-e9f6f57cd00c.png)

The only thing we should do, is update our linter configuration to handle new JSX transform and then don't have to import React: https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html#eslint